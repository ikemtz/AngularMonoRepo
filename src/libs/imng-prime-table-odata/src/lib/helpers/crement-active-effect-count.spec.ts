import {
  decrementActiveEffectCount,
  incrementActiveEffectCount,
} from './crement-active-effect-count';

describe('incrementActiveEffectCount', () => {
  it('should work', () => {
    const result = incrementActiveEffectCount({
      activeEffectCount: 9,
      prop1: 'ðŸ˜ˆ',
    });
    expect(result).toMatchSnapshot();
  });
});
describe('decrementActiveEffectCount', () => {
  it('should work', () => {
    const result = decrementActiveEffectCount({
      activeEffectCount: 19,
      prop1: 'ðŸ˜ˆ',
    });
    expect(result).toMatchSnapshot();
  });
  it('should handle zero', () => {
    const result = decrementActiveEffectCount({
      activeEffectCount: 0,
      prop1: 'ðŸ˜ˆ',
    });
    expect(result).toMatchSnapshot();
  });
  it('should handle below zero', () => {
    const result = decrementActiveEffectCount({
      activeEffectCount: -1,
      prop1: 'ðŸ˜ˆ',
    });
    expect(result).toMatchSnapshot();
  });
});
