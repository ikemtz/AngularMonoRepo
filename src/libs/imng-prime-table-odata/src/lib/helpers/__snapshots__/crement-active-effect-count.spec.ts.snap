// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`decrementActiveEffectCount should handle below zero 1`] = `
Object {
  "activeEffectCount": 0,
  "prop1": "ðŸ˜ˆ",
}
`;

exports[`decrementActiveEffectCount should handle zero 1`] = `
Object {
  "activeEffectCount": 0,
  "prop1": "ðŸ˜ˆ",
}
`;

exports[`decrementActiveEffectCount should work 1`] = `
Object {
  "activeEffectCount": 18,
  "prop1": "ðŸ˜ˆ",
}
`;

exports[`incrementActiveEffectCount should work 1`] = `
Object {
  "activeEffectCount": 10,
  "prop1": "ðŸ˜ˆ",
}
`;
